erpnext/buying/doctype/purchase_order/purchase_order.js

```diff
@@ -24,6 +24,19 @@ frappe.ui.form.on("Purchase Order", {
 	},
 });
 
+frappe.ui.form.on("Purchase Order Item", {
+	item_code: function(frm) {
+		frappe.call({
+			"method": "get_last_purchase_rate",
+			"doc": cur_frm.doc,
+			callback: function(r, rt) {
+				cur_frm.dirty();
```

Comment:
Avoid using `cur_frm` in code. Use `frm` instead of `cur_frm`.

Reply:
Done!